src/Illuminate/Foundation/Exceptions/views/4xx.blade.php

```diff
@@ -0,0 +1,11 @@
+@extends('errors::minimal')
+
+@php
+    $statusCode = $exception->getStatusCode();
```

Comment:
How do we know this method always exists on the exception?

Reply:
We know this because it's a 4xx error page, which only renders when a status code exists in the 400-499 range.